runtime/jvmti/jvmtiExtensionMechanism.c

```diff
@@ -377,6 +377,18 @@ static const jvmtiParamInfo jvmtiGetCarrierThread_params[] = {
 };
 #endif /* JAVA_SPEC_VERSION >= 19 */
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+/* (jvmtiEnv *jvmti_env) */
+static const jvmtiParamInfo jvmtiVMCheckpoint_params[] = {
+	{ "jni_env", JVMTI_KIND_IN_PTR, JVMTI_TYPE_JNIENV, JNI_FALSE },
+	{ "thread", JVMTI_KIND_IN, JVMTI_TYPE_JTHREAD, JNI_FALSE }
+};
+static const jvmtiParamInfo jvmtiVMRestore_params[] = {
+	{ "jni_env", JVMTI_KIND_IN_PTR, JVMTI_TYPE_JNIENV, JNI_FALSE },
+	{ "thread", JVMTI_KIND_IN, JVMTI_TYPE_JTHREAD, JNI_FALSE }
```

Comment:
Either include the trailing comma (like line 877) or don't, but please be consistent.

Reply:
https://github.com/eclipse-openj9/openj9/commit/5768c727965bfb39506877857277d72b3e11d107